runtime/docker/docker.go

```diff
@@ -229,15 +230,15 @@ func (c *DockerRuntime) DeleteNet(ctx context.Context) (err error) {
 }
 
 // CreateContainer creates a docker container
-func (c *DockerRuntime) CreateContainer(ctx context.Context, node *types.NodeConfig) (err error) {
+func (c *DockerRuntime) CreateContainer(ctx context.Context, node *types.NodeConfig) (interface{}, error) {
```

Comment:
@networkop 
can you please add a docstring here exaplaining why `interface{}` got added to the signature?

Reply:
done.

Reply:
I've added it to the runtime interface instead